internal/provider/harness_base_resource.go

```diff
@@ -96,6 +96,11 @@ type ContainerMountModel struct {
 	ReadOnly    types.Bool   `tfsdk:"read_only"`
 }
 
+type ContainerLayerModel struct {
+	Source types.String `tfsdk:"source"`
+	Target types.String `tfsdk:"target"`
```

Comment:
technically `Destination` -> `Target` is a breaking change, but we don't use this anywhere yet and in my head it makes significantly more sense to warrant the breaking change rename

Reply:
Big +1